app/views/coronavirus_landing_page/components/landing_page/_live_stream_section.html.erb

```diff
@@ -1,23 +1,28 @@
-<% if details.show_live_stream? %>
+<section class="covid__topic-wrapper">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: details.live_stream["title"],
+    heading_level: 2,
+    margin_bottom: 6,
+    padding: true,
+    border_top: 2
+  } %>
+
   <%= render partial: 'coronavirus_landing_page/components/landing_page/live_stream', locals: {
     details: details,
```

Comment:
Does this need to be passed in anymore?

Reply:
It's used to get today's date on line 30 of `_live_stream.html.erb`.